src/saltext/vmware/states/roles.py

```diff
@@ -0,0 +1,161 @@
+# SPDX-License-Identifier: Apache-2.0
+import json
+import logging
+
+import saltext.vmware.modules.roles as roles_module
+import saltext.vmware.utils.connect as connect
+import saltext.vmware.utils.drift as drift
+
+log = logging.getLogger(__name__)
+
+try:
+    from pyVmomi import vim
+
+    HAS_PYVMOMI = True
+except ImportError:
+    HAS_PYVMOMI = False
+
+
+__virtualname__ = "vcenter_roles"
+__proxyenabled__ = ["vcenter_roles"]
+
+
+def __virtual__():
+    if not HAS_PYVMOMI:
+        return False, "Unable to import pyVmomi module."
+    return __virtualname__
+
+
+def config(name, config, service_instance=None, profile=None):
+    """
+    Get/Set roles configuration based on drift report.
```

Comment:
**suggestion** This should read something like ```Set vCenter role configuration.``` -- it may be happening due to something on a drift report, but it's not necessary. One can simply use this to enforce their roles configuration.